src/openfl/display/Stage.hx

```diff
@@ -374,12 +375,6 @@ class Stage extends DisplayObjectContainer implements IModule {
 		window.onTextEdit.add (onTextEdit.bind (window));
 		window.onTextInput.add (onTextInput.bind (window));
 		
-		if (window.id > -1) {
-			
-			onWindowCreate (window);
-			
-		}
-		
```

Comment:
This is to avoid creating the renderer twice right?

Reply:
Yes, as mentioned in the relevant commit message :)